functions/Get-DbaDatabase.ps1

```diff
@@ -287,7 +287,6 @@ function Get-DbaDatabase {
                 ($_.Owner -in $Owner -or !$Owner) -and
                 ($_.state -ne 6 -or !$OnlyAccessible)
             }
-
```

Comment:
This empty lines helps a bit to see that getting the "backed_info" is finished and we are now starting to build the $inputObject. Maybe put it back in...